modules/data.remote/R/Prep_AGB_IC_from_geotiff.R

```diff
@@ -0,0 +1,41 @@
+#' Extract above ensemble ground biomass density from pre-existing GeoTIFF file for the SDA workflow.
```

Comment:
Ok, so presumably this isn't a generic approch to reading any AGB geotiff (of which there are many in the world), but something for working with some specific product? That needs to be well documented, including being clear in the function name.

Reply:
fixed.